app/styles/_shame.scss

```diff
@@ -8,13 +8,22 @@
   }
 }
 
+.au-c-toolbar__group--row.comments-row {
+  > ul {
+    margin-left: 0;
+    > .au-o-grid__item {
+      padding-left: 0;
+    }
+  }
+}
```

Comment:
Nitpick: I think these can be replaced by using some of the spacing utils on the correct element?

https://appuniversum.github.io/ember-appuniversum/?path=/story/utilities-spacing--page